src/components/modal/v2/parts/views/NI/Content.jsx

```diff
@@ -55,22 +47,44 @@ export const NI = ({
                                 </li>
                             ))}
                         </ul>
-                        {footer.map(content => {
-                            const line = content.map(item => {
-                                if (Array.isArray(item)) {
-                                    const [text, link] = item;
-                                    return (
-                                        <a target="__blank" href={link}>
-                                            {text}
-                                        </a>
-                                    );
-                                }
-                                return <span>{item}</span>;
-                            });
-                            return <li className="content__footer-item">{line}</li>;
-                        })}
-                        <li className="content__footer-item">{renderProductListLink()}</li>
-                    </ul>
+                        <ul>
+                            {footer.map(content => {
+                                const line = content.map(item => {
+                                    if (Array.isArray(item)) {
+                                        const [text, link] = item;
+                                        return (
+                                            <a target="__blank" href={link}>
+                                                {text}
+                                            </a>
+                                        );
+                                    }
+                                    return <span>{item}</span>;
+                                });
+                                return <li className="content__footer-item">{line}</li>;
+                            })}
+                            {useServerData()?.views?.length > 1 ? (
+                                <li className="content__footer-item">
+                                    <ProductListLink openProductList={openProductList}>
+                                        {linkToProductList}
+                                    </ProductListLink>
+                                </li>
+                            ) : (
+                                <Fragment />
+                            )}
```

Comment:
Since the function is at the top of the file, let's replace this with:
```suggestion
                           <li className="content__footer-item">{renderProductListLink()}</li>
```